lib/systems/platforms.nix

```diff
@@ -500,7 +500,7 @@ rec {
   # based on:
   #   https://www.mail-archive.com/qemu-discuss@nongnu.org/msg05179.html
   #   https://gmplib.org/~tege/qemu.html#mips64-debian
-  mips64el-qemu-linux-gnuabi64 = (import ./examples).mips64el-linux-gnuabi64 // {
+  mips64el-qemu-linux-gnuabi64 = (import ./examples.nix { inherit lib; }).mips64el-linux-gnuabi64 // {
```

Comment:
```suggestion
  mips64el-qemu-linux-gnuabi64 = {
```

Reply:
Oh that was just plain wrong. We shouldn't be done that. I must have missed it in code review.

Reply:
applied your suggestion but i didn't test it other than `:p lib`

Reply:
Good enough for me. If this breaks something else, that deserves a different fix than reverting this.